File: SingleAPIExample/go/VerifyPayment.go
package main

import (
	"context"
	"fmt"

	"github.com/splitit/splitit.sdks.go"
)

func VerifyPayment(ipn string) {
	ctx := context.TODO()

	verifyResp, _, err := apiClient.InstallmentPlanApi.InstallmentPlanVerifyPayment(
		ctx,
		splitit.VerifyPaymentRequest{
			InstallmentPlanNumber: ipn,
		},
	)
	if err != nil {
		fmt.Printf("InstallmentPlanVerifyPayment call failed: %s\n", err)
		return
	}
	if verifyResp.IsPaid {
		fmt.Printf("Plan is payed for %f\n", verifyResp.OriginalAmountPaid)
	} else {
		fmt.Printf("Plan wasn't payed\n")
	}

}

File: SingleAPIExample/TSNode/VerifyPayment.ts
import { InstallmentPlanApi } from 'splitit-sdk-nodejs'

let planApi: InstallmentPlanApi

export async function verifyPayment(installmentPlanNumber: string) {
  const { body: verifyResponse } = await planApi.installmentPlanVerifyPayment({ installmentPlanNumber })
}
